development/paystack.tsx

```diff
@@ -69,6 +71,11 @@ const Paystack: React.ForwardRefRenderFunction<React.ReactNode, PayStackProps> =
   const dynamicSplitString = dynamicSplitObjectIsValid(split) ? `split: ` + JSON.stringify(split)  + `,` : ''; // should only send split for dynamic multi-account split with the correct split object as defined
   //Sometimes, you can't determine a split configuration until later in the purchase flow. With dynamic splits, you can create splits on the fly. This can be achieved by passing a split object
 
+  const planCodeString = plan ? `ref: '${plan}',` : ''; // should only send plan with the predefined plan_code as generated on paystack dashboard if present, else if blank, it will be ignored.
```

Comment:
@opmat `refNumberString` already returns `ref:`, passing  `planCodeString` might not override `refNumberString` and end up causing problems. 

Will there be a case where refNumberString is not needed?

Reply:
@just1and0 thank you for the quick catch. This has been corrected and pushing an updating now